change/@microsoft-fast-element-a347a5ee-bd7e-4c8b-afd8-cc9eb668e804.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Looks like one thing did get missed - since this is just for the docs, we don't want to rev the packages. We'll want this to be a `none` type.